packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -27,65 +24,94 @@ export const stylesBySize = {
   },
 };
 
-export const getContainerStyles = (props: StyleProps): SerializedStyles => css`
-  position: absolute;
-  top: 0;
-  right: 0;
-  height: 100%;
-  width: ${props.size !== 'scale'
-    ? // In case we're using a specific size, we want it to be used until there's no space left.
-      // In such scenario, we want the modal to be as wide as possible, but using the shared indentation width size.
-      `min(${
-        stylesBySize[props.size].width
-      }, calc(100% - ${PORTALS_CONTAINER_INDENTATION_SIZE})) !important`
-    : stylesBySize.scale.width};
-  display: flex;
-  flex-direction: column;
-  background-color: ${customProperties.colorSurface};
-  box-shadow: 0px 0px 40px 0px rgba(0, 0, 0, 0.1);
-  outline: 0;
-  transform: translate3d(
-    ${props.size !== 'scale' ? stylesBySize[props.size].width : '30px'},
-    0,
-    0
-  );
-  transition: transform ${stylesBySize[props.size].transitionTime}ms ease;
+const overlayShow = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
 `;
 
-export const getOverlayStyles = (props: StyleProps): SerializedStyles => css`
+// we can't use Overlay from Radix-ui because it doesn't appear in DOM when Dialog.Root is rendered with `modal={false}`
```

Comment:
Just trying to better understand the issue: why do we need to use `modal={false}`?

Reply:
Thanks Carlos!
Otherwise, when the modal is open, `aria-hidden` is applied to all other elements on the page, preventing the use of the project selector, language selector, navbar, and profile context menu.

Reply:
Oh! I see. Thanks for the explanation 👍